frontend/components/CancelFightConfirmation.js

```diff
@@ -0,0 +1,104 @@
+import React, { useEffect, useState } from 'react';
+import Api from './util/Api';
+import FormHeaderLink from './util/FormHeaderLink';
+import { Flex, Center, Button, Text, ScrollView, Heading } from 'native-base';
+import { Link, Navigate, useLocation } from "react-router-native";
+import { FontAwesome } from '@expo/vector-icons';
+
+const CancelFightConfirmation = () => {
+
+    const props = useLocation({});
+    const [cancelled, setCancelled] = useState(false);
+    useEffect(() => {
+        console.log(props);
+    }, []);
+
+    const cancelFight = () => {
+        Api.delete('/matches/cancel/' + props.state.fightId //change to delete after back update
```

Comment:
We may remove this comment now
```suggestion
        Api.delete('/matches/cancel/' + props.state.fightId
```